public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,2019 @@
+
+    * Raise technical issues on the [interest-restriction-return](https://github.com/hmrc/interest-restriction-return/issues) issues page
+
+    # Changelog
+
+    You can find the change log on the [interest-restriction-return](https://github.com/hmrc/interest-restriction-return/wiki) wiki page.
+
+    # Versioning
+
+    When an API changes in a way that is backwards-incompatible, we increase the version number of the API.
+
+    See our [reference guide](/api-documentation/docs/reference-guide#versioning) for more on versioning.
+
+    # Errors
+
+    We use standard HTTP status codes to show whether an API request succeeded or not. They are usually in the range:
+
+      - 200 to 299 if it succeeded, including code 202 if it was accepted by an API that needs to wait for further action
+      - 400 to 499 if it failed because of a client error by your application
+      - 500 to 599 if it failed because of an error on our server
+
+    Errors specific to each API are shown in the Endpoints section, under Response. See our [reference guide](https://developer.service.hmrc.gov.uk/api-documentation/docs/reference-guide#errors) for more on errors.
+servers:
+  - url: https://test-api.service.hmrc.gov.uk
+    description: Sandbox
+  - url: https://api.service.hmrc.gov.uk
+    description: Production
+components:
+  parameters:
+    acceptHeader:
+      name: Accept
+      in: header
+      style: simple
+      description: Specifies the response format and the [version](/api-documentation/docs/reference-guide#versioning) of the API to be used.
+      schema:
+        type: string
+        enum:
+          - application/vnd.hmrc.1.0+json
+      required: true
+    contentTypeHeader:
+      name: Content-Type
+      in: header
+      style: simple
+      description: Specifies the format of the request body, which must be JSON.
+      schema:
+        type: string
+        enum:
+          - application/json
+      required: true
+    authorizationHeader:
+      name: Authorization
+      in: header
+      style: simple
+      description: An [OAuth 2.0 Bearer Token](/api-documentation/docs/authorisation/user-restricted-endpoints) with the `write:interest-restriction-return` scope.
+      schema:
+        type: string
+        example: Bearer bb7fed3fe10dd235a2ccda3d50fb
+      required: true
+  securitySchemes:
+    user-restricted:
+      type: oauth2
+      description: HMRC supports OAuth 2.0 for authenticating User-restricted API requests
+      flows:
+        authorizationCode:
+          authorizationUrl: https://api.service.hmrc.gov.uk/oauth/authorize
+          tokenUrl: https://api.service.hmrc.gov.uk/oauth/token
+          refreshUrl: https://api.service.hmrc.gov.uk/oauth/refresh
+          scopes:
+            write:interest-restriction-return: Write Interest Restriction Return
```

Comment:
scope value should be `write:interest-restriction-return: Allow write access to Interest Restriction Return data`